test/table-content-with-issues.md

```diff
@@ -9,5 +9,5 @@
 | * emphasis*                  | {MD037} |
 | __strong __                  | {MD037} |
 |  ` code`                     | {MD038} |
-| [link ](https://example.com) | {MD039} |
-| [link]()                     | {MD042} |
+| [link ](https://example.com) | {MD039} {MD059} |
```

Comment:
Could you expand the table rows above so the columns are consistent? I like this and want to keep the new test case.